CompassTests/Shared/TestCompass.swift

```diff
@@ -24,7 +24,7 @@ class TestCompass: XCTestCase {
     let expectation = self.expectationWithDescription("Parse arguments")
     let url = NSURL(string: "compassTests://profile:testUser")!
 
-    Compass.parse(url) { route, arguments in
+    Compass.parse(url) { route, arguments, _ in
```

Comment:
I think you would always resolve it in your code but you don't need it in those tests, I added another test that actually does need the variable. The method that you call to navigate in your application can have it optional.